tasks/last_boot_time.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# who -b prints out the last boot time in the format:
+#           system boot  2018-09-05 05:06
+# the awk statement extracts just the date + time
+who -b | awk '{print $3 " " $4}'
```

Comment:
This doesn't work well on macOS.

Reply:
`last -1 -F reboot` would be ideal, but `-F` doesn't work everywhere (macOS for instance). Might use it anyway, and accept lower granularity on macOS.